pos_default_payment_method/views/pos_config_view.xml

```diff
@@ -0,0 +1,13 @@
+<?xml version="1.0" encoding="utf-8"?>
+<odoo>
+    <record id="view_pos_config_form" model="ir.ui.view">
+        <field name="name">pos.config.form.view default payment method</field>
+        <field name="model">pos.config</field>
+        <field name="inherit_id" ref="point_of_sale.view_pos_config_form"/>
+        <field name="arch" type="xml">
+            <field name="journal_ids" position="after">
+                <field name="default_payment_method_id" />
```

Comment:
Maybe put this field inside a ```group``` tag. Actually we don't see any label so it's really confusing.